### PR TITLE
Task/gv 1155 slider for calculator level

### PR DESCRIPTION
**Pull request by Anchello**

### CODE REVIEW

File: packages/shared/components/calculator-level/calculator-slider/calculator-slider.tsx [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+import Slider from "rc-slider";
+import * as React from "react";
+import NumberFormat from "react-number-format";
+
+class _CalculatorSlider extends React.PureComponent<Props, State> {
+  state = {
+    value: this.props.defaultValue
+  };
+
+  static defaultProps = {
```
Comment (pavelzubov) [inline packages/shared/components/calculator-level/calculator-slider/calculator-slider.tsx]: Don't use defaultProps.
Use `const { min, max, title, step = 1 , valueSuffix } = this.props;` instead it.

Comment (Anchello) [reply packages/shared/components/calculator-level/calculator-slider/calculator-slider.tsx]: Done

**Comment by Anchello** [comment]
I fixed the styles of the slider

**Comment by pavelzubov** [comment]
image

**Comment by Anchello** [comment]
I fixed the styles of the slider again

**Comment by pavelzubov** [comment]
image
Just a little bit more))

**Comment by Anchello** [comment]
I fixed the styles a little bit :)

**Comment by pavelzubov** [comment]
image
No)

**Comment by Anchello** [comment]
Fixed the indent by 2px

**Comment by pavelzubov** [comment]
60096458-aead7e80-9759-11e9-8c81-6af2099d63ff copy
I think it's not 2 px.
60096458-aead7e80-9759-11e9-8c81-6af2099d63ff copy
It's near 6 px.
You can always watch it in Sketch.

**Comment by pavelzubov** [comment]
I think it's not 2 px.
2px
It's near 6 px.
60096458-aead7e80-9759-11e9-8c81-6af2099d63ff copy
You can always watch it in Sketch.

**Comment by Anchello** [comment]
pixel
I have MacBook without Retina and on my screen it looks like that (after fixed by 2px)

**Comment by pavelzubov** [comment]
I looked in the sketch and he showed me about 10px (instead 13px in your commit).
Please, use Sketch)
And please set 10px and we should close this task. We hang on it for a veeeeeeeery long time.

**Comment by Anchello** [comment]
I set 10px, but on my screen it looks like this
Снимок экрана 2019-06-26 в 15 40 36
Снимок экрана 2019-06-26 в 15 42 15

**Pull request merged**